### PR TITLE
fix: demo site link and beautify loading

### PR DESCRIPTION
**Pull request by leo220yuyaodog**
https://user-images.githubusercontent.com/71440988/190289494-4de020f7-f016-454a-b579-e1b672e1213b.mp4

### CODE REVIEW

File: src/components/CasdoorCard/index.js [hunk 1]
```diff
@@ -1,20 +1,74 @@
...
+    cursor: "pointer",
+    borderRadius: "20px",
+  };
   useEffect(() => {
-    setLink(localStorage.getItem("CasdoorLink"));
+    axios.get("https://oa.casbin.com/api/is-mainland-ip").then((response) => {
+      if (response.data === true) {
+        setLink("https://door.casdoor.com/");
+      } else {
+        setLink("https://door.casdoor.org/");
```
Comment (Selflocking) [inline src/components/CasdoorCard/index.js:26]: setLink like this:
image
will help reduce the number of requests

File: src/components/CasdoorCard/index.js [hunk 2]
```diff
@@ -1,20 +1,74 @@
...
+        scrolling="no"
+      />
+      <div style={maskStyle} hidden={!mouseState.state || !props.mask} onClick={() => {window.open(link + props.src);}}>
+        <Link
+          className="button button--secondary button--lg"
+          style={{marginTop: "50%", marginRight: "3rem", marginLeft: "3rem"}}>
+          <Translate>Online Demo</Translate>
+        </Link>
+      </div>
+      <div hidden={loading} style={{
```
Comment (Selflocking) [inline src/components/CasdoorCard/index.js:62]: image

image

In https://deploy-preview-339--iridescent-baklava-0d7550.netlify.app/docs/overview
I removed the `hidden` attribute to see the effect, and found that it was not good

**Review by Selflocking — CHANGES_REQUESTED** [review]

File: src/components/CasdoorCard/index.js (hunk 2 above)
Comment (leo220yuyaodog) [reply src/components/CasdoorCard/index.js]: Right, I have resolved it. plz review

File: src/components/CasdoorCard/index.js [hunk 3]
```diff
@@ -1,20 +1,70 @@
...
+    zIndex: "1",
+    height: "100%",
+    width: "100%",
+    background: "rgba(0,0,0,0.4)",
+    cursor: "pointer",
+    borderRadius: "20px",
+  };
   useEffect(() => {
-    setLink(localStorage.getItem("CasdoorLink"));
+    setLink((localStorage.getItem("CasdoorLink")));
```
Comment (Selflocking) [inline src/components/CasdoorCard/index.js:22]:
```suggestion
    setLink(localStorage.getItem("CasdoorLink"));
```
extra parentheses

**Review by Selflocking — APPROVED** [review]

**Pull request closed without merging**